plugins/modules/digital_ocean_load_balancer.py

```diff
@@ -415,13 +417,25 @@
 
 
 class DOLoadBalancer(object):
+
+    # Regions which use 'size' versus 'size_unit'
+    size_regions = ["ams2", "nyc2", "sfo1"]
```

Comment:
Just a nit, but you _could_ use a `set` here:
```
size_regions = {"ams2", "nyc2", "sfo1"}

Reply:
Good call; will change.